### PR TITLE
refactor(service): fix RSSHub env values

### PR DESCRIPTION
**Pull request by voryzen**
RSSHub had some weird http basic auth values in the environment variables.
even when they were uncommented, they did nothing.

replaced it with a valid variable (TZ)

### CODE REVIEW

File: roles/rsshub/templates/docker-compose.rsshub.yml.j2 [hunk 1]
```diff
@@ -13,9 +13,7 @@ services:
     networks:
       - traefik_network
     environment:
-      # OPTIONAL: http basic auth
-      # - HTTP_BASIC_AUTH_NAME={{ default_username }}
-      # - HTTP_BASIC_AUTH_PASS="{{ default_password }}"
+      - TZ=Etc/UTC
```
Comment (denis-ev) [inline roles/rsshub/templates/docker-compose.rsshub.yml.j2:16]: We got a variable for this 

```suggestion
      - TZ={{ common_timezone }}
```

**Review by denis-ev — CHANGES_REQUESTED** [review]

**Comment by voryzen** [comment]
Forgot about that var.

Good catch

**Pull request merged**